README.md

```diff
@@ -6,75 +6,168 @@
-avoid an immediate `Killed: 9`.
+Usage:
+  ks [command]
+
+Available Commands:
+  apply       Apply local configuration to remote cluster
+  delete      Delete Kubernetes resources described in local config
+  diff        Display differences between server and local config, or server and server config
+  env         Manage ksonnet environments
+  generate    Expand prototype, place in components/ directory of ksonnet app
+  ...
+```
 
 ## Quickstart
 
-```console
-# Include <ksonnet.git>/lib in ksonnet/jsonnet library search path.
-# Can also use explicit `-J` args everywhere.
-% export KUBECFG_JPATH=/path/to/ksonnet/lib
+Here we provide a shell script that shows some basic ksonnet features in action. You can run this script to deploy and update a basic web app UI, via a Kubernetes Service and Deployment. This app is shown below:
+
+<p align="center">
+<img alt="guestbook screenshot" src="/docs/img/guestbook.png" style="width:60% !important;">
+</p>
+
+Note that we will not be implementing the entire app in this quickstart, so the buttons will not work!
+
+**Minimal explanation is provided here, and only basic ksonnet features are shown---this is intended to be a quick demonstration.** If you are interested in learning more, see [Additional Documentation](#additional-documentation).
+
+### Prerequisites
+* *You should have access to an up-and-running Kubernetes cluster (**maximum version 1.7**).* Support for Kubernetes 1.8 is under development.
 
-# Show generated YAML
-% ks show -o yaml -f examples/guestbook.jsonnet
+  If you do not have a cluster, [choose a setup solution](https://kubernetes.io/docs/setup/) from the official Kubernetes docs.
 
-# Create resources
-% ks apply -f examples/guestbook.jsonnet
+* *You should have `kubectl` installed.* If not, follow the instructions for [installing via Homebrew (MacOS)](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos) or [building the binary (Linux)](https://kubernetes.io/docs/tasks/tools/install-kubectl/#tabset-1).
 
-# Modify configuration (downgrade gb-frontend image)
-% sed -i.bak '\,gcr.io/google-samples/gb-frontend,s/:v4/:v3/' examples/guestbook.jsonnet
-# See differences vs server
-% ks diff -f examples/guestbook.jsonnet
+* *Your `$KUBECONFIG` should specify a valid `kubeconfig` file*, which points at the cluster you want to use for this demonstration.
 
-# Update to new config
-% ks apply -f examples/guestbook.jsonnet
+### Script
+
+Copy and paste the script below to deploy the container image for a basic web app UI:
+
+```
+# Start by creating your app directory
+ksonnet init quickstart
+cd quickstart
+
+# Autogenerate a basic manifest
+ksonnet generate deployment-exposed-with-service guestbook-ui \
+--name guestbook \
+--image alpinejay/dns-single-redis-guestbook:0.3 \
+--type ClusterIP
+
+# Deploy your manifest to your cluster
+ksonnet apply default
+
+# Set up an API proxy so that you can access the guestbook service locally
```

Comment:
What's your opinion on removing this entirely and asking them to visit their server? I feel that they should have this information based on the pre-reqs.

Reply:
I want to avoid blocking users who aren't familiar with services/bastion hosts/etc. `LoadBalancer` doesn't work on all clusters (only AWS/GCP/Azure, *and* it's billable), so I opted for API proxy as a relatively agnostic way of getting them to the webpage